models/categories/categories.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const DataModel = require('../memory-data-model.js');
+
+class Categories extends DataModel {
+  constructor() {
+    super();
+    this.schema = {
+      id: { required: true },
+      name: { required: true },
+    };
```

Comment:
This can go out side the class and call it just schema. Then in the parenthesis by `super` you can put schema in there.